plugins/kyverno.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kyverno
+spec:
+  version: v1.1.6
+  homepage: https://github.com/nirmata/kyverno
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/nirmata/kyverno/releases/download/v1.1.6/kyverno-cli_v1.1.6_linux_x86_64.tar.gz
+    sha256: 61560c57ec5eb7f7c9da0d3b8331d90c02b593f8a2a86db86714c7e34f8eb91d
+    files:
+    - from: kyverno
```

Comment:
Please also install the `LICENSE` file along with your executable.
(same in other places)

I see that the license is already part of your tarball. The easiest way to install everything that's in there is to remove the `files` section, as it defaults to `{from: "*", to: "."}`.